bleep-core/src/scala/bleep/packaging/createJar.scala

```diff
@@ -0,0 +1,54 @@
+package bleep.packaging
+
+import bleep.{model, RelPath}
+
+import java.io.ByteArrayOutputStream
+import java.nio.file.attribute.FileTime
+import java.nio.file.{Files, Path}
+import java.util.jar.{Attributes, JarEntry, JarOutputStream, Manifest}
+import scala.collection.mutable
+
+object createJar {
+  private val epochTime = FileTime.fromMillis(0)
```

Comment:
epoch 0 tends to cause issues; https://github.com/sbt/sbt/pull/6254
sbt and Bazel use 2010.

So `1262304000000`, not `0`.
```suggestion
  private val epochTime = FileTime.fromMillis(1262304000000)
```

Reply:
nice, thanks!